svea_vision/scripts/sidewalk_pc_to_og.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import rospy
+import tf2_ros
+from tf2_sensor_msgs.tf2_sensor_msgs import do_transform_cloud
+from sensor_msgs.msg import PointCloud2
+from nav_msgs.msg import OccupancyGrid
+
+import numpy as np
+
+np.float = float    # NOTE: Temporary fix for ros_numpy issue; check #39
+import ros_numpy
+
+
+def load_param(name, value=None):
+    if value is None:
+        assert rospy.has_param(name), f'Missing parameter "{name}"'
+    return rospy.get_param(name, value)
+
+def replace_base(old, new) -> str:
+    split_last = lambda xs: (xs[:-1], xs[-1])
+    is_private = new.startswith('~')
+    is_global = new.startswith('/')
+    assert not (is_private or is_global)
+    ns, _ = split_last(old.split('/'))
+    ns += new.split('/')
+    return '/'.join(ns)
+
+
+class SidewalkPointCloudToOccupancyGrid:
+    
+    def __init__(self):
+        try:
```

Comment:
Good to use try here?

If exception then the node will spin (still alive) but certain functions/features might not work as expected.

As opposed to crashing entire node.

Reply:
This was the old version, I just merged the new version, it signals shutting down the node in case of an exception:
https://github.com/KTH-SML/svea_vision/blob/093d29a347c1efdeecf4ab8742f484c8768f7d19/svea_vision/scripts/sidewalk_mapper.py#L120-L123